BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -570,18 +622,6 @@ fileprivate struct SingleChildLayoutHost: Layout {
     }
 }
 
-// Used for elements with a single child that requires no custom layout
-fileprivate struct PassthroughLayout: SingleChildLayout {
-
-    func measure(in constraint: SizeConstraint, child: Measurable) -> CGSize {
-        child.measure(in: constraint)
-    }
-
-    func layout(size: CGSize, child: Measurable) -> LayoutAttributes {
-        LayoutAttributes(size: size)
-    }
-
-}
```

Comment:
I wonder if some portion of this optimization - which, I think, improves on `SingleChildLayout` by introducing caching - could be introduced to all `SingleChildLayout` adopters, perhaps by updating `SingleChildLayoutHost`?

Reply:
The trick here is that we can skip the layout entirely; because the child is always exactly the same size as the parent – I don't think we can do that for other non-passthrough layouts

Reply:
Though perhaps: This could take an optional `Layout`, which would then allow us to alternatively pass through vs perform the layout. Would be a good follow up to play with